lib/init.js

```diff
@@ -14,7 +14,10 @@ var isFile = require('file-exists');
 var isDirectory = require('is-directory').sync;
 var isGitURL = require('is-git-url');
 var isGitHub = function (input) { return input.match(/^(?!\.)[A-Za-z0-9\-_]+\/[A-Za-z0-9\-_]+$/g); };
-var isGit = function (input) { return isGitURL(input) || isURL(input); };
+var isGit = function (input) {
+    return isGitURL(input) || isURL(input) ||
+        (input.startsWith('git@github.com:') && isGitHub(input.substring('git@github.com:'.length)));
```

Comment:
@howardroark also note this change I made, maybe should have been in a separate PR